fakeprod/gcloud-sqlproxy.yaml

```diff
@@ -11,6 +11,7 @@ cloudsql:
       port: 5432
 rbac:
   create: true
+  pspEnabled: true
```

Comment:
Should this be `false`?

Reply:
Good eye -- I'll actually remove this addition. fakeprod is so far behind, I don't think we would be able to ever redeploy. (ticket to retire fake prod - https://broadworkbench.atlassian.net/browse/DR-3443)